treeherder/perf/stats.py

```diff
@@ -315,6 +293,58 @@ def interpret_effect_size(delta):
         return "large"
 
 
+def interpret_cles_direction(cles, pvalue_threshold=PVALUE_THRESHOLD):
+    greater_rev = None
+    if cles is None:
+        return "CLES cannot be interpreted", greater_rev
+    if cles > pvalue_threshold:
+        greater_rev = "base"
+        return f"{cles:.0%} chance a base value > a new value", greater_rev
+    if cles < pvalue_threshold:
+        greater_rev = "new"
+        return f"{1 - cles:.0%} chance a new value > base value", greater_rev
+    return "CLES cannot be interpreted", greater_rev
```

Comment:
With these changes, we'd also need to change this code: https://github.com/mozilla/treeherder/blob/a829e16f1b78090e1d736267c063a536f657259c/treeherder/perf/stats.py#L385